src/gevent/monkey.py

```diff
@@ -1057,14 +1057,18 @@ def patch_all(socket=True, dns=True, time=True, select=True, thread=True, os=Tru
 
 
 def main():
+    import runpy
```

Comment:
Please do not move the import. It was purposefully *after* monkey-patching had been done.